.github/workflows/deploy-pipeline.yml

```
@@ -0,0 +1,411 @@
+      uses: actions/checkout@v4
+
+    - name: Setup SSH
+      uses: webfactory/ssh-agent@v0.7.0
+      with:
+        ssh-private-key: ${{ secrets.PRODUCTION_SSH_KEY }}
+
+    - name: Create Production Backup
+      run: |
+        ssh -o StrictHostKeyChecking=no ${{ secrets.PRODUCTION_USER }}@${{ secrets.PRODUCTION_HOST }} << 'EOF'
+          cd /opt/church-management
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)